operations/user/usersOperation.js

```diff
@@ -0,0 +1,198 @@
+const Operation = require('../operation');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs')
+const userService = require('../../services')
+const config = require('config');
+const middleware = require('../../middleware/authenticateJWT')
+
+class UserOperation extends Operation {
+	constructor() {
+		super();
+		this.userService = userService;
+	}
+
+	async login({
+		email,
+		password
+	}) {
+		const {
+			SUCCESS,
+			ERROR,
+			VALIDATION_ERROR,
+			NOT_FOUND
+		} = this.outputs;
+		try {
+			const user = await this.userService.userServices.findOneUser({
+				email
+			})
+			if (!user)
+				return this.emit(SUCCESS, {
+					message: 'Email does not exist'
+				})
+
+			if (!(await bcrypt.compare(password, user.password)))
+				return this.emit(SUCCESS, {
+					message: 'Password is not correct'
+				})
```

Comment:
make sure that you type correctly the if statement as mentioned in the README Rules